### PR TITLE
fix logo, images loading and fix react errors issue 384

### PR DESCRIPTION
**Pull request by RasenGUY**
# Description
fixed the image loading issue by just addressing the compilation warnings and fix logo. 

### Fixes (image not loading #384)
Warning: Invalid DOM property `fill-rule`. Did you mean `fillRule`?
-in path (at GithubRepoCard.js:28)
    - developerFolio\src\components\githubRepoCard\GithubRepoCard.js
        - replaced all fill-rule attributes in js file to fillRule    

./src/components/header/Header.js
  Line 76:13:  The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid
    - CHANGED href attribute of parent a tag of Toggleswitch component to from <a href="#"> to <a href="# ">

Greetings, time traveller. We are in the golden age of prefix-less CSS, where Autoprefixer is no longer needed for your stylesheet.
./src/components/githubProfileCard/GithubProfileCard.scss (./node_modules/css-loader/dist/cjs.js??ref--6-oneOf-5-1!./node_modules/postcss-loader/src??postcss!./node_modules/resolve-url-loader??ref--6-oneOf-5-3!./node_modules/sass-loader/dist/cjs.js??ref--6-oneOf-5-4!./src/components/githubProfileCard/GithubProfileCard.scss)
Warning
    - edit browserlist in package.json for production and development
    "production": [
        ">0.3%",
        "not ie 11",
        "not dead",
        "not op_mini all"
        ],
        "development": [
        ">0.3%",
        "not ie 11",
        "not dead",
        "not op_mini all"
        ]

logo-fix 
- src/components/header/header.scss 66:3
  - font-feature-settings: "liga","clig off";
  - .header .menu-icon <-header.scss->
    - inline-block is ignored due to the float. If 'float' has a value other than 'none', the box is floated and 'display' is treated as 'block'
      - commented it out since the code was unecessarcy according to the linter

image not loading fix
Seems like some of the images for the different files weren't loading because they were corrupt. strange since this fixed required another another approach in my own build a while back. but the fix entailed just replacing the images with new (same) ones.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: src/components/header/Header.scss [hunk 1]
```diff
@@ -63,6 +63,7 @@
 .header .logo-name {
   font-family: "Agustina Regular";
   font-weight: bold;
+  font-feature-settings: "liga","clig off";
```
Comment (naveen521kk) [inline src/components/header/Header.scss:66]: What does this do?

Comment (RasenGUY) [reply src/components/header/Header.scss:66]: Those are the original settings of the font, seems to fix the logo issue kart chick was talking about. I think someone removed it with previous PR

**Comment by saadpasta** [comment]
Please rebase with master

**Comment by RasenGUY** [comment]
> Please rebase with master

I think this issue shoeld be closed, since naveen fixed the issues by rolling back some commits, if im not mistaken

**Pull request closed without merging**